major: start a big renovation (#22)
Alzymologist/alzymologist.github.io: content/info/software.md

+++
title = "Software development"
date = 2024-01-13
+++

> ## Contract software development

Reviewer: maybe instead
Software development and consulting

Author: ```suggestion
## Software development and consulting
```

> We offer fullstack contract software development. Main areas of expertise are:

Reviewer: and then

> We offer fullstack contract software development.

We offer full-stack contract software development and consulting services.

Author: ```suggestion
We offer full-stack contract software development and consulting services. Main areas of expertise are:
```

- Embedded electronics

> - Blockchain development (mostly Substrate/Polkadot)

Reviewer: > (mostly Substrate/Polkadot)

похоже лишнее уточнение

Author: ```suggestion
- Blockchain development
```

- Nonconventional payment systems
- Distributed network systems
- Process control and data acquisition
- Secure cryptographic systems

> - Data science/analysis

Reviewer: ```suggestion
- Data science/analysis
- DevOps/SecOps
```

> Preffered stack used in-house includes JS, Rust, Python, C, C++, PHP, Kotlin, R

Reviewer: ```suggestion
Preferred stack used in-house includes JS, Rust, Python, C, C++, PHP, Kotlin, R
```
